tooling/lib/updated.js

```diff
@@ -4,16 +4,17 @@
 
 const debug = require('debug')('tooling:updated');
 const _ = require('lodash');
+
 const {listDependents} = require('./dependencies');
 const {last} = require('./version');
 const {diff} = require('./git');
 
 /**
  * Lists all of the updated packages in the repo
  * @param {Object} options
- * @param {boolean} options.dependents if true, also includes dependents of
+ * @param {boolean=} options.dependents if true, also includes dependents of
```

Comment:
It's JSDoc for optional.
VSCode has a setting that does semantic checking on javascript files using their TSLinter `javascript.implicitProjectConfig.checkJs`. It uses JSDoc or TSLint config to populate the required arguments for a function your calling.
So it for `updated` TSLint would always throw a red line under wherever you called it because `npm` wasn't being included in the function call.